### ISSUE TITLE
A11y_WindowsUILibraryv2.4_HomeScreen_RadialGradientBrush_Screenreader: NVDA is not announcing the heading like "Radial Gradient brush" and the description present under the heading

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/microsoft-ui-xaml#3151]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_WindowsUILibraryv2.4;#A11yTCS;#A11yMAS;#DesktopApp;#MAS1.3.1;#Windows UI Library;#STP;#NVDA;#Win32;

Environment Details:
Application Name: Windows UI Library v2.4
Version: Version 2.4
OS: Windows10

Additional Details:
MAS References:MAS1.3.1

**Repro Steps:**

1. Install and open the Windows UI Library v2.4 application
2. Navigate to List Item pane on the left side of the screen.
3. Navigate to "Radial Gradient brush" list item under "New Control" then navigate to its heading start nvda and check the announcement

**Actual:**
NVDA is not announcing the heading like "Radial Gradient brush" and the description present under the heading with any key combination

**The issue is observed for all the heading of the list items.**

**Issue not observed with Narrator**

**Expected:**
NVDA should announce the heading like "Radial Gradient brush" and the description present under the heading with any key combination

**User Impact:**
Screen reader users will not receive the complete information.

**Attachment link for Reference:**
[3151_A11y_WindowsUILibraryv2.4_HomeScreen_RadialGradientBrush_Screenreader.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5090236/3151_A11y_WindowsUILibraryv2.4_HomeScreen_RadialGradientBrush_Screenreader.zip)

**Comment by StephenLPeters** [comment]
@YuliKl can you investigate?

**Comment by YuliKl** [comment]
Because this issue is not reproing with Narrator, and because this is a test app rather than a shipping product, I believe we should not attempt to fix this bug. The issue is not related to RadialGradientBrush, which is the product actually being shipped. Closing.

**Comment by Shivamd1** [comment]
@YuliKl  If the issue is specific to the screen reader NVDA then we need to file an external bug against the NVDA
and this bug should not be closed by product team .So confirm us whether this is specific to screen reader NVDA then we will log an external bug.

**Comment by YuliKl** [comment]
@Shivamd1 you had stated that
> Issue not observed with Narrator

I can confirm, the app works as expected with Narrator. The WinUI team will not be fixing problems with NVDA so this issue does not belong in our repo. I closed the issue because I don't believe it to be actionable for us.
